strobogrammatic-and-palindrome.html

```diff
@@ -1,74 +1,87 @@
+
+    <section class="container flex justify-center items-center mt-[5%]">
+      <div class="border-2 rounded-[2%] border-slate-800/[0.5] w-6/12 shadow-2xl ">
+        <div class="dark:text-slate-400 text-center mt-[3%] text-lg "><b> &#9432 info </b></div>
+        <p class="m-[3%] p-[2%] dark:text-slate-400">A <b>strobogrammatic</b> number is a number that looks the same when rotated
+          180 degrees (looked at upside down).
+          <br>
+          <br>
+          A <b>Palindrome</b> number is a number (such as 16461) that remains the same when its digits are reversed.
+        </p>
+      </div>
+    </section>
+
+   </div>
   </body>
 
   <script>
-    const submitButton = document.querySelector("#submit");
-    submitButton.addEventListener("click", function(event) {
-      event.preventDefault();
-      const input = document.querySelector("input");
-      const strobogrammatic = document.querySelector("#strobogrammatic");
-      const palindrome = document.querySelector("#palindrome");
-      const result = document.querySelector("#result");
-      result.classList.remove("hidden");
-      const number = input.value;
-      const strobogrammaticResult = isStrobogrammatic(number.toString());
-      const palindromeResult = isPalindrome(number);
-      strobogrammatic.innerHTML = strobogrammaticResult ? '&#9989;' : '&#10060;';
-      palindrome.innerHTML = palindromeResult ? '&#9989;' : '&#10060;';
-    });
+  const submitButton = document.querySelector("#submit");
+  submitButton.addEventListener("click", function (event) {
+    event.preventDefault();
+    const input = document.querySelector("input");
+    const strobogrammatic = document.querySelector("#strobogrammatic");
+    const palindrome = document.querySelector("#palindrome");
+    const result = document.querySelector("#result");
+    result.classList.remove("hidden");
+    const number = input.value;
+    const strobogrammaticResult = isStrobogrammatic(number.toString());
+    const palindromeResult = isPalindrome(number);
+    strobogrammatic.innerHTML = strobogrammaticResult ? '&#9989;' : '&#10060;';
+    palindrome.innerHTML = palindromeResult ? '&#9989;' : '&#10060;';
+  });
 
-    const isPalindrome = (str) => str === [...str].reverse().join('');
+  const isPalindrome = (str) => str === [...str].reverse().join('');
 
-    const isStrobogrammatic = num => {
-      const map = new Map([['6', '9'], ['9', '6'], ['0', '0'], ['1', '1'], ['8', '8'], ['2','2']]);
+  const isStrobogrammatic = num => {
+    const map = new Map([['6', '9'], ['9', '6'], ['0', '0'], ['1', '1'], ['8', '8'], ['2', '2']]);
 
-      for (let i = 0, j = num.length - 1; i <= j; i++, j--) {
-        if (!map.has(num[i]) || map.get(num[i]) !== num[j]) {
-          return false;
-        }
+    for (let i = 0, j = num.length - 1; i <= j; i++, j--) {
+      if (!map.has(num[i]) || map.get(num[i]) !== num[j]) {
+        return false;
       }
+    }
 
-      return true;
-    };
+    return true;
+  };
```

Comment:
It was well indented before, a tab was removed here which puts this in the same level with the `<script>` tag.